apps/common-app/src/examples/RuntimeTests/tests/TestsOfTestingFramework.test.tsx

```diff
@@ -21,20 +21,28 @@ import {
   waitForNotify,
   clearRenderOutput,
 } from '../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
-import { Snapshots } from './Animations.snapshot';
+import { Snapshots } from './TestsOfTestingFramework.snapshot';
 import { ComparisonMode } from '../ReanimatedRuntimeTestsRunner/types';
 
 const AnimatedComponent = () => {
   const widthSV = useSharedValue(0);
-  const ref = useTestRef('AnimatedComponent');
+  const ref = useTestRef('BrownComponent');
+  const ref2 = useTestRef('GreenComponent');
 
-  const animatedStyle = useAnimatedStyle(() => {
+  const animatedStyle1 = useAnimatedStyle(() => {
     callTracker('useAnimatedStyleTracker');
     return {
       width: withTiming(widthSV.value, { duration: 500 }, callTrackerFn('withTimingTracker')),
     };
   });
 
+  const animatedStyle2 = useAnimatedStyle(() => {
+    callTracker('useAnimatedStyleTracker');
```

Comment:
Yes, exactly. I added `skip` decorator to these tests, since, as discussed async, these tests of testing framework may be confusing and they should be skipped by default.